bindings/python/vara-feature/pybind_FeatureModel.cpp

```diff
@@ -19,6 +19,13 @@ void init_feature_model_module(py::module &M) {
       .def_property_readonly(
           "path",
           [](const vf::FeatureModel &FM) { return FM.getPath().string(); })
+      .def_property(
+          "commit",
+          [](const vf::FeatureModel &FM) { return FM.getCommit().str(); },
+          [](vf::FeatureModel &FM, std::string &Commit) {
```

Comment:
Use as string value and move it or a `const &` but value should work better here.